054-AzureLoadTesting/Coach/README.md

```diff
@@ -0,0 +1,70 @@
+# What The Hack - Azure Load Testing - Coach Guide
+
+## Introduction
+Welcome to the coach's guide for the Azure Load Testing What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.  
+
+NOTE: If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+## Duration
+This hack is estimated to take 3 days with 4 hours each day.
+
+## Preparation
```

Comment:
This belongs under the new "Coach Prerequisites" section in the template!  I don't think it needs its own section header.  Just add the content where the template says you should.

Reply:
Done